bitcoin/src/pow.rs

```diff
@@ -390,6 +400,43 @@ impl U256 {
 
     const ONE: U256 = U256(0, 1);
 
+    /// Creates a `U256` from an prefixed hex string.
+    fn from_hex(s: &str) -> Result<Self, PrefixedHexError> {
+        let stripped = if let Some(stripped) = s.strip_prefix("0x") {
+            stripped
+        } else if let Some(stripped) = s.strip_prefix("0X") {
+            stripped
+        } else {
+            return Err(MissingPrefixError::new(s).into());
+        };
+        Ok(U256::from_hex_internal(stripped)?)
+    }
+
+    /// Creates a `CompactTarget` from an unprefixed hex string.
```

Comment:
In f019e24f1f4c58f85177d5d77647dcbf23941b02:

This doccomment is just wrong, but that's fine, it's not a public function .